flashcards/backend/conftest.py

```diff
@@ -43,10 +43,20 @@ def category(db):
     """
     name = "test category"
     description = "test category description"
-
     return Category.objects.create(name=name, description=description)
 
 
+@pytest.fixture
+def difficulty_level(db):
```

Comment:
You have almost identical fixture in line 192 wouldn't be better to use or modify that one to comply with DRY.

Reply:
My mistake, I forgot to remove that one in line 192. It's gone now :smile: Thanks for noticing!
I ran black but then forgot to run it again after I merged latest changes from master. It's updated now.
As per mypy we need to install additional [type stub package](https://pypi.org/project/types-factory-boy/) so that Factory Boy types could be recognized. I'd install it in another task, so would suggest to leave it for now.

Reply:
The problem with difficulty level was handled in that way that now difficulty level value is a unique integer and name is connected to it by using LazyAttribute. So we avoid the situation that same name gets somehow assigned to different values. Then we pass only 'value' attribute to `django_get_or_create` in the factory and it works fine

Reply:
Ready to go